### PR TITLE
VULA-3875 - paginate adding courses

### PR DESCRIPTION
**Pull request by NgoniChoga**

### CODE REVIEW

File: src/java/org/sakaiproject/component/app/scheduler/jobs/ResetContentTypes.java [hunk 1]
```diff
@@ -104,15 +105,15 @@ public void execute(JobExecutionContext arg0) throws JobExecutionException {
...
 
 		for(int x=0; x<=pages; x++) {
 			if(x == pages ) {
-				first = x*100;
+				first = x*SITE_BATCH_SIZE;
 				last = first+remainder;
 			} else {
-				first = x*100;
+				first = x*SITE_BATCH_SIZE;
 				last = first+99;
```
Comment (smarquard) [inline src/java/org/sakaiproject/component/app/scheduler/jobs/ResetContentTypes.java:117]: first+SITE_BATCH_SIZE-1

File: src/java/org/sakaiproject/component/app/scheduler/jobs/UCTCourseSiteOwners.java [hunk 2]
```diff
@@ -113,15 +114,15 @@ private void addCourseOwners() {
...
 
 		for(int i=0; i<=pages; i++) {
 			if(i == pages ) {
-				first = i*100;
+				first = i*SITE_BATCH_SIZE;
 				last = first+remainder;
 			} else {
-				first = i*100;
+				first = i*SITE_BATCH_SIZE;
 				last = first+99;
```
Comment (smarquard) [inline src/java/org/sakaiproject/component/app/scheduler/jobs/UCTCourseSiteOwners.java:126]: last= first+SITE_BATCH_SIZE-1

File: src/java/org/sakaiproject/component/app/scheduler/jobs/ResetContentTypes.java (hunk 1 above)
Comment (TurRil) [reply src/java/org/sakaiproject/component/app/scheduler/jobs/ResetContentTypes.java]: not the result of the calculation - replace the line 

last = first+99;

with:

last= first+SITE_BATCH_SIZE-1

File: src/java/org/sakaiproject/component/app/scheduler/jobs/UCTCourseSiteOwners.java (hunk 2 above)
Comment (TurRil) [reply src/java/org/sakaiproject/component/app/scheduler/jobs/UCTCourseSiteOwners.java]: not the result of the calculation - replace the line 

last = first+99;

with:

last= first+SITE_BATCH_SIZE-1

**Review by TurRil — COMMENTED** [review]
Changes to be made.

**Review by TurRil — APPROVED** [review]

**Pull request merged**